frontend/public/components/modals/tolerations-modal.tsx

```diff
@@ -148,26 +154,28 @@ const TolerationsModal = withHandlePromise((props: TolerationsModalProps) => {
                         title={operators[operator]}
                       />
                     ) : (
-                      <input
-                        type="text"
-                        className="pf-v5-c-form-control"
-                        value={operator}
-                        readOnly
-                      />
+                      <span className="pf-v6-c-form-control pf-m-readonly">
+                        <input type="text" value={operator} readOnly />
+                      </span>
                     )}
                   </div>
                   <div className="clearfix visible-sm visible-xs" />
                   <div className="col-md-3 col-sm-5 col-xs-5 toleration-modal__field">
                     <div className="toleration-modal__heading hidden-md hidden-lg text-secondary text-uppercase">
                       {t('public~Value')}
                     </div>
-                    <input
-                      type="text"
-                      className="pf-v5-c-form-control"
-                      value={value}
-                      onChange={(e) => change(e, i, 'value')}
-                      readOnly={!isEditable(toleration) || operator === 'Exists'}
-                    />
+                    <span
+                      className={classnames('pf-v6-c-form-control', {
+                        'pf-m-readonly': !isEditable(toleration) || operator === 'Exists',
```

Comment:
Pull `!isEditable(toleration) || operator === 'Exists'` out to a `const` so it's not duplicated.